gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/cockpit/command/handler/V4ApiCommandHandler.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.rest.api.service.cockpit.command.handler;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.gravitee.cockpit.api.command.Command;
+import io.gravitee.cockpit.api.command.CommandHandler;
+import io.gravitee.cockpit.api.command.CommandStatus;
+import io.gravitee.cockpit.api.command.v4api.V4ApiCommand;
+import io.gravitee.cockpit.api.command.v4api.V4ApiPayload;
+import io.gravitee.cockpit.api.command.v4api.V4ApiReply;
+import io.gravitee.rest.api.model.UserEntity;
+import io.gravitee.rest.api.service.UserService;
+import io.gravitee.rest.api.service.cockpit.services.V4ApiService;
+import io.gravitee.rest.api.service.common.GraviteeContext;
+import io.reactivex.rxjava3.core.Single;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Ashraful Hasan (ashraful.hasan at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@Component
+public class V4ApiCommandHandler implements CommandHandler<V4ApiCommand, V4ApiReply> {
```

Comment:
added "authenticateAs(user);" function in the handler. 

fyi, the background of this task: The goal is to create a V4 API definition when aAPIM trial is provisioned from Cockpit. Not sure whether you would be able to view this ticket : https://gravitee.atlassian.net/browse/COC-304

Reply:
thank you for the explanation, I now understand why this service is not as complex as the one as mentioned.
we just have to keep in mind that it can only be used in the case of a trial and not for the same as `DeployModelCommandHandler`